### PR TITLE
Add notes about system_auth RF configuration to the documentation

### LINKED ISSUE #1358: system_auth table RF is not extended during scale out?
When you add more nodes to a Scylla cluster, I'm not sure the operator expand the RF for system_auth table, as the docs ask to do. 
(I'm pretty sure I've already opened a bug about it, but couldn't find it)

### PR DESCRIPTION
**Pull request by rzetelskik**
**Description of your changes:** As per https://opensource.docs.scylladb.com/stable/operating-scylla/procedures/cluster-management/add-node-to-cluster.html, users having ScyllaDB configured with `authenticator` set to `PasswordAuthenticator`, need to manually set `system_auth`'s RF after every topology change. This PR adds a note mentioning that to our docs. It also merges scale out / scale down parts of the doc to avoid repetition.

**Which issue is resolved by this Pull Request:**
Resolves #1358

### CODE REVIEW

File: docs/source/generic.md [hunk 1]
```diff
@@ -273,19 +273,31 @@ To change it simply remove the secret. Operator will create a new one. To pick u
 
 To set up monitoring using Prometheus and Grafana follow [this guide](monitoring.md).
 
-## Scale Up
+## Scale out / scale down
```
Comment (tnozicka) [inline docs/source/generic.md:276]: up / down? (so these are equivalent) 
`Scaling ScyllaCLusters`?

**Comment by tnozicka** [comment]
/hold
(for #1418)

File: docs/source/generic.md [hunk 2]
```diff
@@ -273,19 +273,31 @@ To change it simply remove the secret. Operator will create a new one. To pick u
...
 ```
 
+:::{note}
+If you configured Scylla with `authenticator` set to `PasswordAuthenticator`, you need to manually configure the replication factor of the `system_auth` keyspace after every scaling operation.
+
+```console
+kubectl -n scylla exec -it pods/simple-cluster-us-east-1-us-east-1a-0 -c scylla -- cqlsh -e "ALTER KEYSPACE system_auth WITH REPLICATION = {'class' : 'NetworkTopologyStrategy', 'us-east-1' : <new_replication_factor>};"
+```
+
+It is recommended to set `system_auth` replication factor to the number of nodes in a datacenter.
```
[the remark this answers is not in the document]
Comment (rzetelskik) [reply docs/source/generic.md:298]: That's a good point actually, maybe it shouldn't touch on scaling at all. Would you rather move this to `Configure scylla`? I've put it in scaling to align with Scylla docs about adding/removing nodes, but you're right that we do that for any SCs with #nodes > 1.

File: docs/source/generic.md [hunk 3]
```diff
@@ -273,19 +273,31 @@ To change it simply remove the secret. Operator will create a new one. To pick u
...
-The operator supports scale up of a rack as well as addition of new racks. To make the changes, you can use:
+The operator supports adding new nodes to existing racks, adding new racks to the cluster, as well as removing both single nodes and entire racks. To introduce the changes, edit the cluster with:
 ```console
-kubectl -n scylla edit ScyllaCluster simple-cluster
+kubectl -n scylla edit scyllaclusters.scylla.scylladb.com simple-cluster
 ```
-* To scale up a rack, change the `Spec.Members` field of the rack to the desired value.
-* To add a new rack, append the `racks` list with a new rack. Remember to choose a different rack name for the new rack.
-* After editing and saving the yaml, check your cluster's Status and Events for information on what's happening:
+* The modify the number of nodes in a rack, update the `members` field of a selected rack to a desired value.
```
Comment (annastuchlik) [inline docs/source/generic.md:282]:
```suggestion
* To modify the number of nodes in a rack, update the `members` field of the selected rack to a desired value.
```

File: docs/source/generic.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (annastuchlik) [reply docs/source/generic.md:298]: You may consider leaving the guideline here and adding there to a more general page if that benefits the user.

**Review by annastuchlik — COMMENTED** [review]
The content looks good; I've left minor comments. 
I can't check how it renders until https://github.com/scylladb/scylla-operator/pull/1418 is merged.

File: docs/source/generic.md [hunk 4]
```diff
@@ -273,19 +273,31 @@ To change it simply remove the secret. Operator will create a new one. To pick u
...
 To set up monitoring using Prometheus and Grafana follow [this guide](monitoring.md).
 
-## Scale Up
+## Scale out / scale down
 
-The operator supports scale up of a rack as well as addition of new racks. To make the changes, you can use:
+The operator supports adding new nodes to existing racks, adding new racks to the cluster, as well as removing both single nodes and entire racks. To introduce the changes, edit the cluster with:
 ```console
-kubectl -n scylla edit ScyllaCluster simple-cluster
+kubectl -n scylla edit scyllaclusters.scylla.scylladb.com simple-cluster
```
[the remark this answers is not in the document]
Comment (rzetelskik) [reply docs/source/generic.md]: done

File: docs/source/generic.md [hunk 5]
```diff
@@ -273,19 +273,31 @@ To change it simply remove the secret. Operator will create a new one. To pick u
...
+* To remove a rack, first scale it down to zero nodes, and then remove it from `.spec.datacenter.racks` list.
+
+Having edited and saved the yaml, you can check your cluster's Status and Events to retrieve information about what's happening:
 ```console
-kubectl -n scylla describe ScyllaCluster simple-cluster
+kubectl -n scylla describe scyllaclusters.scylla.scylladb.com simple-cluster
 ```
 
+:::{note}
+If you configured Scylla with `authenticator` set to `PasswordAuthenticator`, you need to manually configure the replication factor of the `system_auth` keyspace after every scaling operation.
```
[the remark this answers is not in the document]
Comment (rzetelskik) [reply docs/source/generic.md]: done

File: docs/source/generic.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (rzetelskik) [reply docs/source/generic.md]: Added an additional note to `Configure Scylla` section.

File: docs/source/generic.md (hunk 1 above)
Comment (rzetelskik) [reply docs/source/generic.md]: done

File: docs/source/generic.md (hunk 3 above)
Comment (rzetelskik) [reply docs/source/generic.md]: done

File: docs/source/generic.md [hunk 6]
```diff
@@ -273,19 +284,31 @@ To change it simply remove the secret. Operator will create a new one. To pick u
...
+The operator supports adding new nodes to existing racks, adding new racks to the cluster, as well as removing both single nodes and entire racks. To introduce the changes, edit the cluster with:
 ```console
-kubectl -n scylla edit ScyllaCluster simple-cluster
+kubectl -n scylla edit scyllaclusters.scylla.scylladb.com/simple-cluster
 ```
-* To scale up a rack, change the `Spec.Members` field of the rack to the desired value.
-* To add a new rack, append the `racks` list with a new rack. Remember to choose a different rack name for the new rack.
-* After editing and saving the yaml, check your cluster's Status and Events for information on what's happening:
+* To modify the number of nodes in a rack, update the `members` field of the selected rack to a desired value.
+* To add a new rack, append it to the `.spec.datacenter.racks` list. Remember to choose a different rack name for the new rack.
```
Comment (zimnx) [inline docs/source/generic.md:294]:
```suggestion
* To add a new rack, append it to the `.spec.datacenter.racks` list. Remember to choose a unique rack name for the new rack.
```

**Review by zimnx — APPROVED** [review]
one nit

File: docs/source/generic.md (hunk 6 above)
Comment (rzetelskik) [reply docs/source/generic.md]: done

**Review by tnozicka — COMMENTED** [review]
/lgtm

**Comment by rzetelskik** [comment]
I changed the MyST syntax back to recommonmark to decouple from #1418.
/hold cancel

**Comment by tnozicka** [comment]
/lgtm

/hold cancel
(no longer needs #1418)

**Pull request merged**